server/conf/helper/cloud.conf

```diff
@@ -17,6 +17,8 @@ azure.blob.account=devstoreaccount1
 azure.blob.account=${?AZURE_STORAGE_ACCOUNT_NAME}
 azure.blob.container=civiform-local-blob
 azure.blob.container=${?AZURE_STORAGE_ACCOUNT_CONTAINER}
+azure.blob.public_container=civiform-local-blob-public
+azure.blob.public_container=${?AZURE_STORAGE_ACCOUNT_PUBLIC_CONTAINER}
```

Comment:
```suggestion
azure.blob.public_container_name=civiform-local-blob-public
azure.blob.public_container_name=${?AZURE_STORAGE_ACCOUNT_PUBLIC_CONTAINER_NAME}
```

Reply:
@dkatzz We messed up with AWS, but since no one is using Azure yet, can we get away with rename `azure.blob.container` to `azure.blob.private_container_name`. That way it is clear that it private.

Reply:
ok will rename the new vars.. and in a separate PR will rename the existing vars for private.   should be manageable to update the existing staging deployment.

Reply:
Yes, that sounds good to me! Miami Dade hasn't gotten their deployment up yet, so this shouldn't be an issue.

Reply:
done

Will update the private container name env var in a separate PR